projects/smart-classroom.md

```diff
@@ -9,10 +9,20 @@ While taking classes in Ithaca during the summer, participation was a significan
 
 This made me realize classrooms haven't changed except for maybe the projector for a very long time, and utilize no technology.
 
-### Challenge
 
-To make a 'smart classroom' leveraging technology such as beacons and cameras (facial recognition, eye direction, body movements, facial expressions, etc) to change the way participation is graded, teachers get feedback, etc etc
+### Story:
+(For September 23rd)
```

Comment:
The story should be in a project management tool like Trello or Waffle.io. Could you please take it out of here?